R-packages/evalcast/R/plot_coverage.R

```diff
@@ -40,17 +58,17 @@ plot_coverage <- function(scorecards,
     
     
     if (ngrps == 3L) {
-      g <- g + geom_line(aes(color = !!sym(grps[1]))) +
+      g <- g + geom_line(aes(color = as.factor(!!sym(grps[1])))) +
```

Comment:
Starting here is the cruft that I'd like to get rid of. Can you follow the logic in `plot_canonical` (see [here](https://github.com/cmu-delphi/covidcast/blob/256944ce6bdccf83b0878997bc6b72179f333d89/R-packages/evalcast/R/plot_canonical.R#L74) for example) and get rid of this automatic choice garbage?

@JedGrabman assuming such a change doesn't screw up the eval dashboard too much.

Reply:
The eval dashboard uses data processing from evalcast, but has its own frontend, so updating plot functions shouldn't affect it.

Reply:
Do you want me to change the signature to use `facet_rows` and `facet_cols` arguments like `plot_canonical()` does?  Or is there something different?

Reply:
Yeah, I think change the signature. So faceting, and colors exactly as done in `plot_canonical()`.

Reply:
Done.